CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## [15.2.0] - TBD
+### Changed
+* Updated `aws-java-sdk-s3` version to 1.11.728 (was 1.11.505).
```

Comment:
This is OK, you could also just have one line mentioning the property that you changed as it affects all 3 (and maybe some other dependencies in child modules).